.github/workflows/ci_build.yml

```diff
@@ -41,3 +41,8 @@ jobs:
           eval $(opam env)
           cd theories/
           make
+      - name: test parser
+        run: |
+          cd ../
```

Comment:
I don't think you should `cd ../` instead, you should `cd` into `driver`? just `cd driver`. also, you will need to do `eval $(opam env)` to have dune visible.

Reply:
check and add the installation for the dependencies. also remember to update README